tuxemon/states/combat/combat_animations.py

```diff
@@ -44,7 +44,7 @@
 
 sprite_layer = 0
 hud_layer = 100
-TimedCallable = Tuple[Callable, float]
+TimedCallable = Tuple[Callable[[], None], float]
```

Comment:
pretty sure that you should be using `list` here instead of `[]`

Reply:
Good catch, but I think it should be `List`.
`list` is the better alias (since you don't need extra import), but it came with Python 3.9.

Reply:
I tried with List, but mypy was flagging me (mypy 1.6.1), and it looks like the suggested format is:
`TimedCallable = Tuple[partial[None], float]`

Reply:
it was suggested that we retire python 3.8 support, and it might be a good time to do that.  i think the only major distro supporting it is ubuntu 16.04 lts, and there is a new lts now, so im not sure we need to support 3.8.  im in favor of preferring `list` over `List` in this case, and we can deprecate python 3.8 support (and tests)

Reply:
Yes, `list` over `List`, `dict` over `Dict`, `tuple` over `Tuple` and etc., I'm all for it!